client/src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```

Comment:
You deleted this file, so remove all imports of deleted files, delete all unneeded code, and make sure the app is running without problems.